privacyscore/backend/management/commands/rescanscanlist.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (C) 2018 PrivacyScore Contributors
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+import os
+from time import sleep
+
+from django.core.management import BaseCommand
+from django.utils import timezone
+
+from privacyscore.backend.models import Site, ScanList
+from privacyscore.utils import normalize_url
+
+
+class Command(BaseCommand):
+    help = 'Rescan all sites in exisiting ScanList.'
+
+    def add_arguments(self, parser):
+        parser.add_argument('scan_list_id')
+        parser.add_argument('-s', '--sleep-between-scans', type=float, default=0)
+
+    def handle(self, *args, **options):
+        if not ScanList.objects.filter(id=options['scan_list_id']).exists():
+            raise ValueError('Scan List doesnot exist!')
```

Comment:
There's a tiny typo here. "does not".

Make it `objects.get()`, then we don't need to issue the query twice.

Reply:
It will throw `query does not exist` exception . We could either use `.exists( )` or `.first( )`

Reply:
catch the exception then.  But I think that letting the exception bubble up to the user is fine.

Reply:
Ok. I will update it to objects.get( )